scripts/version_calc.py

```
@@ -0,0 +1,312 @@
+    "fix": BumpLevel.PATCH,
+    "perf": BumpLevel.PATCH,
+    "revert": BumpLevel.PATCH,
+}
+
+
+def parse_version(tag: str) -> Version:
+    """Parse a version tag like 'v0.1.0' or '0.1.0' into a Version."""
+    raw = tag.lstrip("v")
+    parts = raw.split(".")
+    if len(parts) != 3:
+        raise ValueError(f"Invalid version tag: {tag}")
```

Ruff (0.15.9)

[warning] Avoid specifying long messages outside the exception class

(TRY003)